### ISSUE TITLE
put ghcjs in npm or the like

### ISSUE THREAD

**Issue by zkessin** [ghcjs/ghcjs#546]
There should be a pre built version of ghcjs. IF you go follow the instructions on stack to install ghcjs it comiles it from scratch which takes a very long time (~2 hours on my MacBook air) if this is a developer's first experience with ghcjs it will be a very negative one. (For reference Elm and Purescript both install from NPM and it takes a minute or less). 

I will say that if I was not using GHCJS for my job that alone would have been enough to convince me 
to go try something else.

**Comment by crocket** [comment]
I fully agree that the compilation is a huge obstacle between me and ghcjs development.

**Comment by zkessin** [comment]
if so it is not documented in the page for ghcjs on the stackage site.

**Comment by crocket** [comment]
Meanwhile, you have https://docs.haskellstack.org/en/stable/ghcjs/
This is slightly better.

**Comment by hamishmack** [comment]
There is also https://github.com/reflex-frp/reflex-platform#reflex-platform which I believe will download binaries for you too.

**Comment by zkessin** [comment]
it does, but it also installs nix which I didn't really want, i just wanted to do something like "apt-get install ghcjs" and have ghcjs work, i am unclear why that is not an option and I have to do a multi hour build to use ghcjs

**Comment by hamishmack** [comment]
> it does, but it also installs nix which I didn't really want, i just wanted to do something like "apt-get install ghcjs" and have ghcjs work, i am unclear why that is not an option and I have to do a multi hour build to use ghcjs

Yep that would be awesome we probably just need someone who knows how to make debian packages and has time to make it happen.  RPM, HomeBrew, MacPorts and MSYS2 would also be cool if anyone has experience with those.

**Comment by zkessin** [comment]
ok, maybe put that on the main page?
